platforms/web/public/locales/en/common.json

```diff
@@ -27,5 +28,6 @@
   "slide_next": "Next slide",
   "slide_previous": "Previous slide",
   "slide_right": "Slide right",
+  "slide_to": "Slide to page {{page}} of {{pages}}",
```

Comment:
Minor; I would go with "Slide to item x of y" or "Go to item x of y". When we wrap it in an `<ul>`, the "x of y" part is not needed in my eyes.

I think we should only use "pages" if there are more items per slide.

Using "pages" as a metaphor, could give the assumption a page can consist of multiple items. Such as

Reply:
I agree!